src/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -412,6 +415,18 @@ public static StatusScreen getStatus(Town town, CommandSender sender) {
 					.append(Component.newline())
 					.append(Component.text(translator.of("status_hover_click_for_more")))),
 				ClickEvent.runCommand("/towny:town reslist "+ town.getName()));
+			
+			// Plots
+			screen.addComponentOf("plotsnewline", Component.newline());
+			TextComponent text = Component.empty();
+			Map<TownBlockType, Integer> cache = town.getTownBlockTypeCache().getCache(TownBlockTypeCache.CacheType.RESIDENTOWNED);

```

Comment:
This is only going to show player-owned plots and not the entire town's plot type collection. CacheType.ALL ought to be used.